.changeset/spotty-toes-help.md

```diff
@@ -0,0 +1,11 @@
+---
+'@graphql-hive/yoga': patch
+---
+
+Align with the latest yoga;
```

Comment:
link to "latest yoga" and maybe specify the version here as well.

Reply:
Updated, good point! Let me know what you think!